pkg/tailfile/tailfile.go

```diff
@@ -9,6 +9,8 @@ import (
 const blockSize = 1024
 
 var eol = []byte("\n")
+
+// ErrNonPositiveLinesNumber the lines number was negative
```

Comment:
"ErrNonPositiveLinesNumber is an error returned if the lines number was negative"